library/src/main/java/com/datatorrent/lib/io/fs/AbstractFileInputOperator.java

```diff
@@ -517,6 +552,18 @@ public void teardown()
   }
 
   @Override
+  public void activate(OperatorContext context)
+  {
+    scanDirService.scheduleAtFixedRate(new DirectoryScannerThread(), 0, scanIntervalMillis, TimeUnit.MILLISECONDS);
```

Comment:
I follow similar design as AbstractJdbcPollInputOperator, and make it periodic to reduce the I/O cost of continuous scan.